Snippets update (#6204)
carbon-language/carbon-lang: docs/design/README.md

```carbon
import Core library "io";

> import Math; //hypothetical package. Do not exist yet.

Reviewer: ```suggestion
// This is a hypothetical package for demonstrating imports.
import Math;
```

Reviewer: (carbon does not have trailing comments)

Author: addressed.
use leading, thanks

Reviewer: There's no plan to make it exist, so I don't think we should say "yet". I provided what I think is a more accurate comment.

Author: @danakj sry that I do not fully caught proposed intent of change.
Addressed now.
please re-review.
thanks.
